docs/manual/manual_throttle.rst

```diff
@@ -0,0 +1,346 @@
+direction the motor will spin when receiving a positive throttle command, only the clockwise or counterclockwise selection matters for that. The importance 
+of the 2D/3D distinction will be covered in the :ref:`throttle_mapping` section.
+
+.. figure:: ../_static/manual_images/throttle/throttle_motor_direction.png
+    :align: center
+    :width: 60%
+    :alt: Motor Direction Parameter
+
+    Motor Direction Parameter in IQ Control Center
+
+.. _throttle_mapping:
+
+Mapping to Throttle
+==============================
+
+As mentioned in the :ref:`throttle_sources` section above some throttle sources, namely IQUART and hobby protocols, send their throttle commands as a number from 0.0 to 1.0, 
+which then must be mapped onto the full -100% to 100% range that a throttle command can take. This 0.0 to 1.0 number is referred to in the rest of this section as the “raw value”. 
+Note this is not the case for DroneCAN, these mappings are not necessary for DroneCAN throttles as they are sent with explicit signs.
+
+The exact nature of this mapping depends on how the Vertiq module is configured. Specifically it depends on how the *FC 2D/3D Mode* and the *Direction* parameters 
+are configured in the IQ Control Center. The *FC 2D/3D Mode* parameter can be set to either *2D* or *3D*. The *Direction* parameter can be set to *3D Clockwise*, 
+*3D Counterclockwise*, *2D Clockwise*, or *2D Counterclockwise*. The images below show these parameters in the IQ Control Center.
+
+.. figure:: ../_static/manual_images/throttle/throttle_fc_mode.png
+    :align: center
+    :width: 60%
+    :alt: FC 2D/3D Mode Parameter
+
+    FC 2D/3D Mode Parameter in IQ Control Center
+
+.. figure:: ../_static/manual_images/throttle/throttle_motor_direction.png
+    :align: center
+    :width: 60%
+    :alt: Motor Direction Parameter
+
+    Motor Direction Parameter in IQ Control Center
+
+For the mapping, all that matters for the *Direction* is whether it is 2D or 3D, the clockwise or counterclockwise selection only affects what direction a positive 
+throttle command means as covered in the :ref:`throttle_direction` section. This means that there are four meaningful combinations of these parameters that will result in 
+different mappings. Each of these combinations and the mapping they result in is covered below.
+
+.. _throttle_2d_2d_mapping:
+
+FC Mode = 2D, Motor Direction = 2D
+************************************
+This is the most commonly used configuration. With this setup, raw values map directly to throttle commands, meaning that raw values cannot map to negative throttle commands 
+and therefore cannot reverse the module. Raw values will only make the module spin in the module’s configured direction.
+
+More explicitly, the simple equation below details how to convert from a raw value to a throttle command percentage with this setup.

```

Comment:
The equation below details the conversion from a raw value to a throttle command percentage under this configuration.